### DISCUSSION TITLE
Bring back the Toolbox from WInforms but updated for maui

### DISCUSSION THREAD

**Discussion by davidbuckleyni** [dotnet/maui#9947] [Ideas]
I loved how in the old days we would see a list of controls in the toolbox on Winforms then we drag the control onto the screen. Could something similar be done for maui but it would drop in some skeleton code in the xmal

**Comment by jsuarezruiz** [comment]
Thanks for the feedback. What would you like to see exactly? The Toolbox works by dragging into XAML code, etc.
![maui-toolbox](https://user-images.githubusercontent.com/6755973/188881096-169c3cb2-6432-43b4-88cc-452742c4d376.gif)

**Reply by davidbuckleyni** [reply]
U no i actually didnt no it did that already @jsuarezruiz WHEN i look at at the min I dont see the controls 

![image](https://user-images.githubusercontent.com/5619229/188887408-5a22b17f-363a-47d6-9cee-2bc33763a2bd.png)

![image](https://user-images.githubusercontent.com/5619229/188887482-d2f25892-5f6e-428e-9b0f-73b27eeb7edb.png)

**Comment by davidbuckleyni** [comment]
@jsuarezruiz  did u see my reply above
